mixins/localize-mixin.md

```diff
@@ -56,27 +58,56 @@ class MyComponent extends LocalizeStaticMixin(LitElement) {
 
 }
 ```
-
 #### Example 2: Dynamically Imported Resources
 
 This approach is better for components with many language resources. By importing them dynamically, only the resources for the requested language are actually fetched and downloaded.
 
 Store your resources in individual files, one for each language:
 ```javascript
 // en.js
-export const val = {
+export default {
   'hello': 'Hello {firstName}!'
 };
 ```
 
-Then dynamically import the matching file:
+**Note:** To avoid accidental imports and errors, your resource files should have a dedicated directory with nothing else in it.
+
+Then create your `config` getter:
+```javascript
+import { LocalizeDynamicMixin } from '@brightspace-ui/core/mixins/localize-dynamic-mixin.js';
+
+class MyComponent extends LocalizeDynamicMixin(LitElement) {
+
+  static get config() {
+    return {
+      importFunc: lang => import(`../lang/${lang}.js`) // Path must be relative!
```

Comment:
Could this not `import('../lang/${lang}.js').val` or `import('../lang/${lang}.js').default` and remove the need for the `exportName` config?

Reply:
Yeeeeesssss.